### PR TITLE
Update observability bundle

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR:

- Updates the observability bundle to not counteract with prometheus-meta-operator

### Testing

- [x] Fresh install works.
- [x] Upgrade from previous version works.

### Checklist

- [x] Update changelog in `CHANGELOG.md`.
- [x] Make sure `values.yaml` is valid.

### CODE REVIEW

File: helm/default-apps-openstack/values.schema.json [hunk 1]
```diff
@@ -402,10 +402,10 @@
                                 }
                             }
                         },
-                        "forceUpgrade": {
```
Comment (marieroque) [inline helm/default-apps-openstack/values.schema.json:405]: Why this change ? Shouldn't it be in alphabetical order ?

Comment (QuentinBisson) [reply helm/default-apps-openstack/values.schema.json:405]: Well probably yes. I'll fix it tomorrow :)

Comment (QuentinBisson) [reply helm/default-apps-openstack/values.schema.json]: Fixed

File: helm/default-apps-openstack/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: default-apps-openstack
 description: A Helm chart defining the preinstalled apps running in all Giant Swarm OpenStack clusters.
 home: https://github.com/giantswarm/default-apps-openstack
-version: 0.8.0
+version: 0.8.1
```
Comment (erkanerol) [inline helm/default-apps-openstack/Chart.yaml:5]: this is updated by the release job. you don't need to change it in this PR.

**Review by erkanerol — APPROVED** [review]

File: helm/default-apps-openstack/Chart.yaml (hunk 2 above)
Comment (QuentinBisson) [reply helm/default-apps-openstack/Chart.yaml]: Oh that's neat. Is it a custom CI job?
